src/plugins/intel_cpu/tests/functional/single_layer_tests/gather.cpp

```diff
@@ -809,15 +1035,23 @@ INSTANTIATE_TEST_SUITE_P(smoke_static_4D_ref16, GatherLayerTestCPU,
                     ::testing::Values(additionalConfig[0])),
                 GatherLayerTestCPU::getTestCaseName);
 
+// TODO
+// This tests checks the Blocked/Short and the Blocked/Long cases if it use reference implementation
+// This checks fails currently because of the next.
+// Currently the Blocked/Short case was implemented in JIT kernel and is not use reference implementation anymore
+// But the Blocked/Long case has not been implemented in the JIT kernel yet.
+// And these tests must be uncommented after its implementation
+/*
```

Comment:
Better to disable tests [here](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp)